extensions/amp-story/1.0/amp-story-interactive-poll.js

```diff
@@ -202,6 +202,19 @@ export class AmpStoryInteractivePoll extends AmpStoryInteractive {
         '.i-amphtml-story-interactive-option-percentage-text'
       ).textContent = `${percentage}%`;
       currOption.setAttribute('style', `flex-grow: ${percentage} !important`);
+
+      if (percentage < 20) {
+        const textContainer = currOption.querySelector(
+          '.i-amphtml-story-interactive-option-text-container'
+        );
+        textContainer.setAttribute(
+          'style',
+          `transform: translateX(${index === 0 ? 27 : -27}%) !important`
```

Comment:
Can we make this `27` a top file constant and call it `MIN_HORIZONTAL_PADDING_{LEFT|RIGHT}` or something like that?

Reply:
Done.